cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -123,6 +125,16 @@
 #   The Recording Service library if found (includes rtirecordingservice,
 #   rtiroutingservice, rtirsinfrastructure, nddscpp2, rtidlc, nddsmetp,
 #   rticonnextmsgc and rtixml2).
+# - ``RTIConnextDDS::cloud_discovery_service_c``
+#   The Cloud Discovery Service library if found (includes rtirecordingservice,
```

Comment:
AFAIK there is not depends on `rtirecordingservice`

Reply:
That's right.

Reply:
Typo, wanted to say `rticlouddiscoveryservice`. Good catch.